HC_rot.f90

```diff
@@ -306,12 +306,12 @@ subroutine BiCGstabMPI(matrix,te,source,count)
     double precision,intent(inout)::te(ix,jx,kx),source(ix,jx,kx)
     double precision residual(ix,jx,kx)
     integer i,j,k,n
-    double precision total_resi,total_source,tmp
+    double precision total_resi(1),total_source(1),tmp
```

Comment:
It is worth noting that these changes are related to compiling with newer gfortran compiler and not hdf5 changes.

Reply:
Was this one of your changes at the beginning because I do not remember making this?